draft-ietf-quic-multipath.md

```diff
@@ -323,13 +323,28 @@ client on a new path, if the server decides to use the new path,
 the server MUST perform path validation ({{Section 8.2 of QUIC-TRANSPORT}})
 unless it has previously validated that address.
 
-If validation succeed, the client can send non-probing, 1-RTT packets
-on the new paths.  In contrast with the specification in
+If validation succeeds, the client can continue to use the path.
+If validation fails, the client MUST NOT use the path and can
+remove any status assiciated to the path initation attempt.
+{{Section 9.1 of QUIC-TRANSPORT}} introduces the concept of
+"probing" and "non-probing" frames. When the  multipath extension
+is negoatiated, the receiption of "non-probing"
+packet on a new path needs to be considered as an path initiation
+attempt that does not impact the path status of any existing
+path. Therefore, any frame can be sent on a new path at any time
+as long as the anti-amplification limits
+({{Section 21.1.1.1 of QUIC-TRANSPORT}}) and the congestion control limits are respected.
```

Comment:
The anti-amplification limit only apply on non-validated paths, right? Also, each of these limits are path-specific and probably deserves stating it.